ovos_stt_http_server/audio_utils.py

```
@@ -0,0 +1,50 @@
+        audio.export(buf, format="wav")
+        buf.seek(0)
+        with wave.open(buf) as wf:
+            sample_rate = wf.getframerate()
+            sample_width = wf.getsampwidth()
+            audio_bytes = wf.readframes(wf.getnframes())
+        return AudioData(audio_bytes, sample_rate, sample_width)
+    except ImportError:
+        raise HTTPException(
+            status_code=501,
+            detail=f"Format '{ext}' requires pydub. Install with: pip install pydub",
+        )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)